src/fu-plugin.c

```diff
@@ -403,11 +404,10 @@ fu_plugin_device_add (FuPlugin *self, FuDevice *device)
 		return;
 	}
 
-	g_debug ("emit added from %s: %s",
-		 fu_plugin_get_name (self),
-		 fu_device_get_id (device));
 	fu_device_set_created (device, (guint64) g_get_real_time () / G_USEC_PER_SEC);
 	fu_device_set_plugin (device, fu_plugin_get_name (self));
+	str = fu_device_to_string (device);
+	g_debug ("emit added from %s: %s", fu_plugin_get_name (self), str);
```

Comment:
This is already being shown from `FuMain` no?  Seems like I see it printed multiple times now